### PR TITLE
No longer support per-level custom blocks

### PR DESCRIPTION
**Pull request by joshlory**
Now that we have Block Pools, that is the primary supported way to add custom blocks to a level.  No production levels should specify their own custom blocks inline.

I moved the "Read more..." tips to the block edit page.

### CODE REVIEW

**Comment by joshlory** [comment]
[Stop supporting per-level custom blocks](https://trello.com/c/U8jOKQOl/249-stop-supporting-per-level-custom-blocks)

File: dashboard/app/models/levels/gamelab_jr.rb [hunk 1]
```diff
@@ -25,8 +25,6 @@
 
 class GamelabJr < Gamelab
   serialized_attrs %w(
-    custom_helper_library
```
Comment (balderdash) [inline dashboard/app/models/levels/gamelab_jr.rb:28]: What happens when we load a level that has an old `custom_helper_library` or `custom_blocks`?

Comment (joshlory) [reply dashboard/app/models/levels/gamelab_jr.rb:28]: The data will still be there in the `properties` hash, but there won't be a getter or setter method for it.

**Review by balderdash — APPROVED** [review]

File: dashboard/app/models/levels/gamelab_jr.rb (hunk 1 above)
Comment (joshlory) [reply dashboard/app/models/levels/gamelab_jr.rb:28]: Sorry, I misunderstood your question.  You're right — there is an issue where these blocks will break.  We talked in person about mapping the `gamelab_draggable` per-level-custom-block to a block-pool-defined-block.

**Pull request closed without merging**